results/results.js

```diff
@@ -1,33 +1,263 @@
+    incrementTimesSeen(sessionVotes, display);
+});
+
+// same thing as session but for total
+totalPicks.forEach(pick => {
+    incrementTimesPicked(totalVotes, pick);
+});
+
+totalDisplay.forEach(display => {
+    incrementTimesSeen(totalVotes, display);
+});
+
+// session info for graphs
+const graphIds = sessionVotes.map(el => el.id);
+const graphSessionPicked = sessionVotes.map(el => el.timesPicked);
+const graphSessionSeen = sessionVotes.map(el => el.timesSeen);
+
+var ctx = document.getElementById('session-results-picked').getContext('2d');
+new Chart(ctx, {
+    type: 'bar',
+    data: {
+        labels: graphIds,
+        datasets: [{
+            label: '# of Votes',
+            data: graphSessionPicked,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.2)',
+                'rgba(54, 162, 235, 0.2)',
+                'rgba(255, 206, 86, 0.2)',
+                'rgba(75, 192, 192, 0.2)',
+                'rgba(153, 102, 255, 0.2)',
+                'rgba(255, 159, 64, 0.2)'
+            ],
+            borderColor: [
+                'rgba(255, 99, 132, 1)',
+                'rgba(54, 162, 235, 1)',
+                'rgba(255, 206, 86, 1)',
+                'rgba(75, 192, 192, 1)',
+                'rgba(153, 102, 255, 1)',
+                'rgba(255, 159, 64, 1)'
+            ],
+            borderWidth: 1
+        }]
+    },
+    options: {
+        scales: {
+            yAxes: [{
+                ticks: {
+                    beginAtZero: true
+                }
+            }]
+        }
+    }
+});
+
+var sesDisplay = document.getElementById('session-results-displayed').getContext('2d');
+new Chart(sesDisplay, {
+    type: 'bar',
+    data: {
+        labels: graphIds,
+        datasets: [{
+            label: '# of Times Seen',
+            data: graphSessionSeen,
+            backgroundColor: [
+                'rgba(255, 99, 132, 0.2)',
+                'rgba(54, 162, 235, 0.2)',
+                'rgba(255, 206, 86, 0.2)',
+                'rgba(75, 192, 192, 0.2)',
+                'rgba(153, 102, 255, 0.2)',
+                'rgba(255, 159, 64, 0.2)'
+            ],
+            borderColor: [
+                'rgba(255, 99, 132, 1)',
+                'rgba(54, 162, 235, 1)',
+                'rgba(255, 206, 86, 1)',
+                'rgba(75, 192, 192, 1)',
+                'rgba(153, 102, 255, 1)',
+                'rgba(255, 159, 64, 1)'
+            ],
+            borderWidth: 1
+        }]
+    },
+    options: {
+        scales: {
+            yAxes: [{
+                ticks: {
+                    beginAtZero: true
+                }
+            }]
+        }
+    }
+});
+
+// all time info for graphs
+const totalGraphIds = totalVotes.map(el => el.id);
+const totalGraphSessionPicked = totalVotes.map(el => el.timesPicked);
+const totalGraphSessionSeen = totalVotes.map(el => el.timesSeen);
+
+var totPick = document.getElementById('total-results-picked').getContext('2d');
```

Comment:
Do you think you could add this as a second bar within the first chart?

Reply:
I was trying to figure out how to do that, but couldn't seem to figure it out. I'm not really sure if chart js has that capability, but if it does I would totally like to do that.